conans/test/integration/cmake_flags_test.py

```diff
@@ -201,7 +201,8 @@ def build(self):
         client.save({"conanfile.py": conanfile % "True"})
         client.run("build", ignore_error=True)
 
-        self.assertIn("Execute 'conan install -g txt' first", client.user_io.out)
+        self.assertIn("self.deps_cpp_info not defined. If you need it for a local command run "
```

Comment:
I am confused why this message has changed. I think the code about this is not related to this PR. I have difficulties comparing some refactors, movement of functions